server/web/server.go

```diff
@@ -15,6 +15,7 @@ type Route interface {
 }
 
 // NewRouter returns a new HTTP router.
+// TODO: use gorilla/mux and friends
```

Comment:
🙏🏻

Reply:
Switched to gorilla/mux here https://github.com/charmbracelet/soft-serve/pull/351/commits/0d7a70361d1035658583c06d67b258baf897b711 https://github.com/charmbracelet/soft-serve/pull/351